definitions/infra-prometheus-redis/golden_metrics.yml

```diff
@@ -0,0 +1,17 @@
+redisConnectionsReceivedPerSecond:
+  title: Number of connections received per second
+  unit: OPERATIONS_PER_SECOND
+  query:
+    select: avg(redis.instance.net.connectionsReceivedPerSecond)
+
+redisCommandsProcessedPerSecond:
+  title: Number of commands processed
+  unit: OPERATIONS_PER_SECOND
```

Comment:
The golden metrics don't need the unit field, so these can be removed

Reply:
done